server/schedulers/utils_test.go

```diff
@@ -50,3 +51,80 @@ func (s *testMinMaxSuite) TestMinDuration(c *C) {
 	c.Assert(minDuration(time.Second, time.Minute), Equals, time.Second)
 	c.Assert(minDuration(time.Second, time.Second), Equals, time.Second)
 }
+
+var _ = Suite(&testScoreInfosSuite{})
+
+type testScoreInfosSuite struct{}
+
+func (s *testScoreInfosSuite) TestSortScoreInfos(c *C) {
```

Comment:
Split it as several `TestXxxx`s please. A single unit test should be as small as possible.